lib/commands/general.js

```diff
@@ -43,18 +43,34 @@ commands.mobileIsToastVisible = async function (opts = {}) {
 };
 
 
-// API to invoke methods defined in Android app
-// Eg: To invoke a single method called foo(), opts will be
-//      [{name:"foo"}]
-// If method takes parameters eg foo(String lol, String bar), opts should be
-//      [{name:"foo", args:["lol","bar"]}]
-// Methods can be chained together with opts looking like
-//      [{name:"foo", args:["lol","bar"]}, {name:"qux"}]
+/**
+ *  API to invoke methods defined in Android app.
+ *  Methods can be chained. Next method will be invoked on output of first method
```

Comment:
Nothing is said aboid void return result
Also, I would expect to see the full list of supported types

Reply:
done